flink-connectors/flink-connector-cassandra/src/test/java/org/apache/flink/streaming/connectors/cassandra/CassandraConnectorITCase.java

```diff
@@ -174,70 +151,18 @@ protected Cluster buildCluster(Cluster.Builder builder) {
         }
     }
 
-    private static class EmbeddedCassandraService {
-        CassandraDaemon cassandraDaemon;
-
-        public void start() throws IOException {
-            this.cassandraDaemon = new CassandraDaemon();
-            this.cassandraDaemon.init(null);
-            this.cassandraDaemon.start();
-        }
-
-        public void stop() {
-            this.cassandraDaemon.stop();
-        }
+    public static CassandraContainer createCassandraContainer(String imageTag) {
+        CassandraContainer cassandra =
+                new CassandraContainer(DockerImageName.parse("cassandra").withTag(imageTag));
+        cassandra.withJmxReporting(false);
+        return cassandra;
     }
 
-    @ClassRule public static final TemporaryFolder TEMPORARY_FOLDER = new TemporaryFolder();
-
     @BeforeClass
-    public static void startCassandra() throws IOException {
-
-        ClassLoader classLoader = CassandraConnectorITCase.class.getClassLoader();
-        File file = new File(classLoader.getResource("cassandra.yaml").getFile());
```

Comment:
can we remove the cassandra.yaml test resource?

Reply:
Yes good catch I removed the code but forgot about the file